src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1260,6 +1257,16 @@ forkEnvironment.setWorkingDir("/data/folder")
 forkEnvironment.setJavaHome("/usr/java/default")
 ```
 
+By default, the forked mode is configured then all workflow tasks are executed in a forked JVM separated from the ProActive Node's JVM.
```

Comment:
By default, all workflow tasks are executed in the forked mode, that is, executed in a forked JVM separated from the ProActive Node's JVM.

(the reader may misunderstand "configured" as "specified in the global configure file", while default global configure not set pa.scheduler.task.fork)